packages/config/tests/normalize/fixtures/props_case/netlify.toml

```diff
@@ -11,3 +10,6 @@ TEST = "test"
 
 [Build.Processing.css]
 bundle = true
+
+[functions]
+Directory = "functions"
```

Comment:
Do we need to support `Directory` case (as opposed to `directory`)?
This is supported (but undocumented) inside `config.build` only at the moment, for backward compatibility.
I am wondering if we would be better off not supporting this with the new syntax.

Reply:
I do see your point for not perpetuating this undocumented feature, but at the same time it could be a bit strange to have some properties case sensitive and others not. I feel that I'd need a bit more context on why this feature was introduced in the first place. Either way, I would suggest that we keep this PR as is, so that we limit it to the scope of the `functions.directory` property, and we have this discussion separately. If we do come to the conclusion that we want to remove it, we can create a separate PR to handle it.

What do you think?

Reply:
> it could be a bit strange to have some properties case sensitive and others not. 

We already do. The `build.*` properties are case-insensitive, but not the other configuration properties (except `build` itself).

> What do you think?

:+1: